### PR TITLE
Define "display list"

### PR DESCRIPTION
**Pull request by notriddle**
Fix a TODO in the Glossary.

---

- [ ] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [ ] These changes fix no issue; there wasn't one filed AFAIK
- [X] These changes do not require tests because reasons

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14100)

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The thread that controls a collection of related web content. This could be thou
 
 ### Display list ###
 
-TODO
+A list of concrete rendering instructions. The display list is post-layout, so all items have absolute pixel positions, and z-index has already been applied, so items later in the display list will always be on top of items earlier in it.
```
Comment (emilio) [inline docs/glossary.md]: Are you sure all positions are absolute? I thought they were relative to the parent stacking context?

Comment (mrobinson) [inline docs/glossary.md]: Items in the display list are indeed relative to their parent stacking context and do not include any transformations applied by parents.

Comment (notriddle) [inline docs/glossary.md]: Fixed.

**Comment by emilio** [comment]
r=me with a squash, thanks for taking care of the glossary :)

**Pull request merged**